enarxbot-assigned

```diff
@@ -0,0 +1,117 @@
+#!/usr/bin/python3
+# SPDX-License-Identifier: Apache-2.0
+
+import bot
+
+import json
+import sys
+import os
+
+if os.environ["GITHUB_EVENT_NAME"] not in ["pull_request_target", "issues"]:
```

Comment:
Is this something that runs per event, so if this task isn't relevant to the event then it exits?

Reply:
Correct. If the event triggering it isn't either a pull request or issue being assigned, we don't care about it.

Similar filtering logic exists for all other actions like this.